.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: ruby
 rvm:
-  - 2.2.3
-before_install: gem install bundler -v 1.10.6
+  - 2.5.1
+  - 2.4.4
+  - 2.3.7
+  - 2.2.10
```

Comment:
We should drop 2.2; upstream support has ended.

Reply:
I don't plan on supporting 2.2, it's just a check to see if it's still compatible for people still on 2.2.